mopidy_spotify/web.py

```diff
@@ -526,3 +549,91 @@ def from_uri(cls, uri):
             return cls(uri, LinkType.PLAYLIST, parts[3], parts[1])
 
         raise ValueError(f"Could not parse {uri!r} as a Spotify URI")
+
+
+def _playlist_edit(web_client, playlist, method, **kwargs):
```

Comment:
because in the test harness, `web_client_mock` is `Mock(spec=web.SpotifyOAuthClient)`. so if these functions were part of SOAC, they would be replaced by mock, and not get called, making them untestable. i didn't want to rewrite web_client_mock to use a "manually mocked" SOAC, since that would require touching a whole lot of preexisting tests (the `test_*` functions often use `.return_value` or `.side_effect`; those would have to all be replaced)